releases/katello/3.16/settings

```diff
@@ -0,0 +1,4 @@
+FULLVERSION='3.16.0-rc4'
```

Comment:
I think by now we're at `3.16.0`

Reply:
We're actually at rc4.1

Reply:
Technically, it's 3.16.0-rc4.1 iirc, but this is adding my work from last week,to be built upon by the person doing the next release.

Reply:
I'll defer to @ehelms if he wants me to update to rc4.1 here or if he'll handle it after this merge